superset/sql_parse.py

```diff
@@ -105,36 +105,39 @@ def __get_full_name(tlist: TokenList) -> Optional[str]:
         return None
 
     @staticmethod
-    def __is_identifier(token: Token):
+    def __is_identifier(token: Token) -> bool:
         return isinstance(token, (IdentifierList, Identifier))
 
-    def __process_tokenlist(self, tlist: TokenList):
+    def __process_tokenlist(self, token_list: TokenList):
+        """
+        Add table names to table set
+
+        :param token_list: TokenList to be processed
+        """
         # exclude subselects
-        if "(" not in str(tlist):
-            table_name = self.__get_full_name(tlist)
+        if "(" not in str(token_list):
+            table_name = self.__get_full_name(token_list)
             if table_name and not table_name.startswith(CTE_PREFIX):
                 self._table_names.add(table_name)
             return
 
         # store aliases
-        if tlist.has_alias():
-            self._alias_names.add(tlist.get_alias())
+        if token_list.has_alias():
+            self._alias_names.add(token_list.get_alias())
 
         # some aliases are not parsed properly
-        if tlist.tokens[0].ttype == Name:
-            self._alias_names.add(tlist.tokens[0].value)
-        self.__extract_from_token(tlist)
+        if token_list.tokens[0].ttype == Name:
+            self._alias_names.add(token_list.tokens[0].value)
+        self.__extract_from_token(token_list)
```

Comment:
It looks like __extract_from_token expects a param of type `Token` but token_list is a `TokenList` is the difference important to note?

Reply:
`TokenList` actually extends `Token`, so technically this is correct (see https://github.com/andialbrecht/sqlparse/blob/master/sqlparse/sql.py#L131-L144 , i.e. `TokenList` just adds a `tokens` property) , but I agree, code readability could be improved here 👍